packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss

```diff
@@ -0,0 +1,77 @@
+@import "@royalnavy/css-framework";
+
+$border-radius: 4px;
```

Comment:
wonder if we should have a common border radius or if that's over the top

Reply:
As we implement a few more components I think common would be good - i'm sure there are probably edge cases though - Lets pick designer brains.